pubspec.yaml

```
@@ -10,12 +10,12 @@ environment:
 
 dependencies:
   analyzer: '>=7.4.0 <9.0.0'
-  build: ^3.0.0
+  build: '>=3.0.0 <5.0.0'
   build_test: ^3.3.0
   dart_style: ^3.0.0
   meta: ^1.15.0
   path: ^1.9.0
-  source_gen: ^3.0.0
+  source_gen: '>=3.0.0 <5.0.0'
```

Shouldn't this be `>=3.1`?

```suggestion
  source_gen: '>=3.1.0 <5.0.0'
```

---

3.0.0 doesn't have the `typeNamed` method, only 3.1.0 added it.

---

Well why didn't you tell me earlier! 😄 

Good point....